config/200-bootstrap.yaml

```diff
@@ -83,6 +83,16 @@ data:
                     pipe:
                       path: /tmp/envoy.admin
         - name: xds_cluster
+          # This keepalive is recommended by envoy docs.
+          # https://www.envoyproxy.io/docs/envoy/latest/api-docs/xds_protocol
+          typed_extension_protocol_options:
+            envoy.extensions.upstreams.http.v3.HttpProtocolOptions:
+              "@type": type.googleapis.com/envoy.extensions.upstreams.http.v3.HttpProtocolOptions
+              explicit_http_config:
+                http2_protocol_options:
```

Comment:
I wonder, there is already a `http2_protocol_options` key in the yaml below, would it clash with this one

Reply:
The `http2_protocol_options` below is deprecated one as per https://github.com/envoyproxy/envoy/issues/15559 and https://github.com/envoyproxy/envoy/pull/15563 updated the doc.

I will remove the deprecated one later. (I will send another PR to clean up with `strict_dns` vs `STRICT_DNS`.)